src/acp/policy/setResourcePolicy.ts

```diff
@@ -34,7 +36,7 @@
  * @param resourceWithAcr The Resource whose Access Control Resource contains Access Policies.
  * @param policy The Policy to insert into the Resource's Access Control Resource.
  * @returns A new Resource equal to the given Resource, but with the given Policy in its Access Control Resource.
- * @since 1.6.0
+ * @since 1.18.0
```

Comment:
Should this change if the change is transparent to the user ?

Reply:
Hahaha, actually, my strongest opinion would be is it worth re-running the CI if we ammend it?